peering/peering.go

```diff
@@ -32,12 +33,25 @@ const (
 
 var logger = log.Logger("peering")
 
-type state int
+type PeeringState uint
+
+func (s PeeringState) String() string {
+	switch s {
+	case PeeringInit:
+		return "init"
+	case PeeringRunning:
+		return "running"
+	case PeeringStopped:
+		return "stopped"
+	default:
+		return "unkown peering state: " + strconv.FormatUint(uint64(s), 10)
+	}
+}
 
 const (
-	stateInit state = iota
-	stateRunning
-	stateStopped
+	PeeringInit PeeringState = iota
+	PeeringRunning
+	PeeringStopped
 )
```

Comment:
Sorry to change my mind but thoses names are bad.
The package is already `peering` so calling thoses would do `peering.PeeringInit`.
Could you pls drop the `Peering` prefix on the type and the constants ?

Reply:
EDIT: pls add `State` in front of all constants, to avoid confusion with `init()` functions.

Reply:
NVM I'll push this